ui/component/sideNavigation/view.jsx

```diff
@@ -87,6 +87,13 @@ const PLAYLISTS: SideNavLink = {
   hideForUnauth: true,
 };
 
+const HISTORY: SideNavLink = {
+  title: 'History',
+  link: `/$/${PAGES.HISTORY}`,
+  icon: ICONS.WATCHHISTORY,
+  hideForUnauth: true,
+};
```

Comment:
@tzarebczan mentioned not wanting to expand the sidebar any further when we brought back Purchases ...